StardewArchipelago/GameModifications/CodeInjections/EntranceInjections.cs

```diff
@@ -51,6 +51,14 @@ public static void Initialize(IMonitor monitor, ArchipelagoClient archipelago, E
             catch (Exception ex)
             {
                 _monitor.Log($"Failed in {nameof(PerformWarpFarmer_EntranceRandomization_Prefix)}:\n{ex}", LogLevel.Error);
+                var targetPosition = new Point(tileX, tileY);

```

Comment:
I don't understand why do this here, instead of within the entrance manager code?